tests/rbd_mirror/rbd_mirror_utils.py

```diff
@@ -315,18 +316,21 @@ def get_mirror_mode(self, *args):
             return mirroring_details.get("mode")
         return None
 
-    def mirror_snapshot_schedule_add(self, poolname, **kwargs):
+    def mirror_snapshot_schedule_add(self, **kwargs):
         """
         This will schedule snapshot based on below kwargs
         Args:
-            poolname:
             **kwargs:
-                imagename : if image name is specified schedule will be done per image if not will done globally
+                poolname: if pool name is specified schedule will be done per pool, if not will be done globally
+                imagename : if image name is specified schedule will be done per image if not will be done per pool or
+                            globally
                 interval : Positional argument need to be specified
```

Comment:
[optional] : Can we please add detail on default value of 1 m

Reply:
Done.